src/main/resources/io/github/railroad/lang/es_es.lang

```diff
@@ -31,6 +31,11 @@ railroad.home.settings.general.github.disconnect=Desconectar
 railroad.home.settings.general.language=Idioma:
 
 railroad.home.settings.appearance.theme=Seleccionar un tema
+railroad.home.settings.appearance.downloadtheme=Download themesE
+railroad.home.settings.appearance.notfound=Themes not foundE
+railroad.home.settings.appearance.download=DownloadE
+railroad.home.settings.appearance.installed=InstalledE
+railroad.home.settings.appearance.preview=PreviewE
```

Comment:
What's going on here?

Reply:
Oops that was just to check it worked properly 😂 
Will translate it now.